### ISSUE TITLE
Arcore app not working on Google Nexus 5X, just bought nexus for testing its running android 8.1.0

### ISSUE THREAD

**Issue by hamayonbaig** [google-ar/arcore-android-sdk#447]
Hi,

I have just bought Google Nexus 5X for testing and developing some arcore apps but what when I installed arcore android app from the play store, its not running. 

App is installed but not showing up in menu and also not showing open button in application manager.

What could be the issue, please help, just bought for testing and its not working.

**Comment by inio** [comment]
ARCore is a service.  You install it and then install an app that uses it.  Try searching for "Just a Line" for example.

**Comment by hamayonbaig** [comment]
So it means that, if I develop an arcore app than the users will need to install arcore app first to make my app work on their devices ?

**Comment by vadim0872** [comment]
Yes, this`s how. Read the doc https://developers.google.com/ar/develop/java/enable-arcore it answer on all your questions.
